website/education/views.py

```diff
@@ -40,8 +46,8 @@ def get_context_data(self, **kwargs) -> dict:
                         "categories": x.categories.all(),
                         "document_count": sum(
                             [
-                                x.summary_set.filter(accepted=True).count(),
-                                x.exam_set.filter(accepted=True).count(),
+                                x.summary_count,
+                                x.exam_count,
                             ]
                             + [
                                 c.summary_set.filter(accepted=True).count()
```

Comment:
They do, otherwise each course spawns a query to count the number of exams and a query to count the number of summaries.

That said, I should probably implement this for the old courses as well. At the very least so that there is 1 query per course instead of 1 per old course.